### PR TITLE
Added Gulp to compile a minified version

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "convergame-engine",
+  "version": "1.2.4",
```
Comment (DivineOmega) [inline package.json]: Might be worth making this 0.0.1 as there is no stable release yet.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "license": "MIT",
```
Comment (DivineOmega) [inline package.json]: Should be GPL3 to match the repo's LICENSE file.

**Comment by DivineOmega** [comment]
Great work, thanks!

I've dropped a couple of comments in the commit that should resolved before this can be merged.

**Comment by jbrooksuk** [comment]
Fixed the whoopsies I made from another project.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "convergame-engine",
+  "version": "0.0.1",
+  "license": "GLP3",
```
Comment (DivineOmega) [inline package.json]: Whoops. :)

Comment (jbrooksuk) [inline package.json]: I don't even have fat fingers haha.

Comment (jbrooksuk) [inline package.json]: I imagine it should be GPLv3?

Comment (DivineOmega) [inline package.json]: Lesser GPL3.

Not sure on the syntax to represent this in `package.json`. I'd assume it is either LGPL3 or LGPLv3.

**Comment by jbrooksuk** [comment]
Done!

**Comment by DivineOmega** [comment]
Awesome. Thank you James. :+1:

**Pull request merged**